the-collab-lab/tcl-73-smart-shopping-list
# Style List component

## Description

Style the `List` page.

## Acceptance Criteria
- [x] Enlarge checkbox and trashcan for accessibility
- [x] Align all info relating to items in two rows, wrap if text gets too long for single line
- [x] Alter instructions on page for a cleaner UI
- [x] Instruct user to sign in if they aren't
- [x] Show the list owner message at the bottom of the `List` and `Manage List` page, make it sticky

## Type of Changes

Styling

## Updates

### Before
![Screenshot 2024-04-01 at 11 04 00 PM](https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/111473039/4e2ae1ea-6d36-4a66-be89-ef7836a29a29)

![Screenshot 2024-04-01 at 11 03 53 PM](https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/111473039/3cb3bc01-7f7a-4077-a217-5bca661d8a81)

![Screenshot 2024-04-01 at 11 07 39 PM](https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/111473039/79ccf663-06b9-49b2-a1bf-2f0ca7166e85)

![Screenshot 2024-04-01 at 11 07 02 PM](https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/111473039/8c3cec5a-b0c3-435e-9c27-ae8a9385b303)

![Screenshot 2024-04-01 at 11 07 29 PM](https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/111473039/6fbc082c-3e2b-4d3b-b6d2-4f8598bb985a)

![Screenshot 2024-04-01 at 11 07 17 PM](https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/111473039/f06654b9-e269-47f8-a2be-bd49934c08b1)



### After
![Screenshot 2024-04-01 at 10 50 36 PM](https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/111473039/5f6421e8-6034-4d69-93bd-2130e3caf4aa)

![Screenshot 2024-04-01 at 10 50 45 PM](https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/111473039/c9cb5cbf-d96d-4c8a-b83c-f802889e291d)

![Screenshot 2024-04-01 at 10 10 49 PM](https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/111473039/678b7b34-ea98-454a-8984-63c7ccdff68e)

![Screenshot 2024-04-01 at 10 11 05 PM](https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/111473039/142e081f-e04f-41f7-b8c4-256e78738dc6)

![Screenshot 2024-04-01 at 11 12 07 PM](https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/111473039/2fc4d2f7-8ebe-4588-b04f-c7921b879d2e)

![Screenshot 2024-04-01 at 11 11 54 PM](https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/111473039/2c011a56-ba37-492e-b3a5-5c339c93f022)





## Testing Steps / QA Criteria
* make sure to test instructions with responsive and desktop views
- `git pull`
- `git checkout rs-style-list-component`
- `npm ci`
- `npm start`
- open an incognito browser, make sure you're signed out and can see the message saying to sign in to use the list page
- sign in, select a list from the `Home` page that has no items in it, navigate to the `List` page and check the styling
- select a list from the `Home` page that has items in it, check the styling on the `List` page 
- Navigate to the `Manage List` page, check the list owner message is at the bottom, just above the nav bar

Note: This is all in light mode, after the light/dark mode toggle issue is merged, this issue will be tested to ensure dark mode work as it should.
